import "RewardConfig.proto";

package EnlivenKillConfig;

message RankReward
{
	required uint32 rank	= 1; //排名
	repeated RewardConfig.Reward reward = 2; //排名奖励 
}

message DailyKillReward
{
	required uint32 index	= 1;  
	repeated RewardConfig.Reward reward = 2; //每日杀敌奖励 
}

message OpenPlatform
{
	required uint32 startday	= 1;  //距离开服天数
	repeated uint32 kill_quota	= 2; //杀敌额度
}

message EnlivenKill
{
	repeated RankReward rank_reward	= 1;  //排行榜奖励
	repeated DailyKillReward daily_kill_reward	= 2;  //每日活跃奖励
	repeated OpenPlatform open_platform	= 3;  //杀敌额度配置
}